mcda/utils/utils_for_main.py

```diff
@@ -1104,8 +1166,12 @@ def _save_output_files(scores: Optional[pd.DataFrame],
     """
     Save output files based of the computed scores, ranks, and configuration data.
     """
+    custom_output_path = os.environ.get('PROMCDA_OUTPUT_PATH')  # check if an environmental variable is set
```

Comment:
You are extracting INPUT variable in one place so far and OUTPUT in 4 places. Do you think it would be a better approach to extract them at class level? Otherwise, if you are extracting them in methods then they will be called repeatedly how many times these methods would be called. Point to note here is that these env variables will not change once set, so a class variable might be something that we can consider here?

Reply:
The paths are now extracted once at the beginning of the script and then shared with all the functions throughout the script. This approach indeed helps avoid redundancy.